ghost/admin/app/components/gh-nav-menu/footer.hbs

```diff
@@ -1,9 +1,16 @@
-<div class="gh-nav-bottom">
+<div class="gh-nav-bottom" {{did-insert this.loadCurrentMRR}}>
     {{#if this.hasThemeErrors}}
-    <button type="button" class="gh-theme-error-toast" {{on "click" (fn this.openThemeErrors null)}}>
-        <span class="gh-notification-title">Your theme contains errors</span>
-        <span class="gh-theme-error-p">Some functionality on your site may be limited &rarr;</span>
+    <button type="button" class="gh-footer-toast gh-theme-error-toast" {{on "click" (fn this.openThemeErrors null)}}>
+        <span class="gh-footer-toast-title gh-notification-title">Your theme contains errors</span>
+        <span class="gh-footer-toast-p">Some functionality on your site may be limited &rarr;</span>
     </button>
+    {{else if (and this.showReferralInvite (not this.hasThemeErrors))}}
```

Comment:
This can be simplified, the `not this.hasThemeErrors` is already handled by the `if/else`